Source/Scene/ImageryLayerCollection.js

```diff
@@ -326,38 +326,20 @@ ImageryLayerCollection.prototype.lowerToBottom = function (layer) {
 var applicableRectangleScratch = new Rectangle();
 
 /**
- * Asynchronously determines the imagery layer features that are intersected by a pick ray.  The intersected imagery
- * layer features are found by invoking {@link ImageryProvider#pickFeatures} for each imagery layer tile intersected
- * by the pick ray.  To compute a pick ray from a location on the screen, use {@link Camera.getPickRay}.
+ * Helper function for `pickImageryLayers` and `pickImageryLayerFeatures`.
  *
  * @param {Ray} ray The ray to test for intersection.
  * @param {Scene} scene The scene.
- * @return {Promise.<ImageryLayerFeatureInfo[]>|undefined} A promise that resolves to an array of features intersected by the pick ray.
- *                                             If it can be quickly determined that no features are intersected (for example,
- *                                             because no active imagery providers support {@link ImageryProvider#pickFeatures}
- *                                             or because the pick ray does not intersect the surface), this function will
- *                                             return undefined.
+ * @return {[QuadtreeTile,TileImagery[],Cartographic]|undefined}  Returns an array containing
+ *                                        pickedTile, imageryTiles, and the pickedLocation.
+ *
+ * @private
  *
- * @example
- * var pickRay = viewer.camera.getPickRay(windowPosition);
- * var featuresPromise = viewer.imageryLayers.pickImageryLayerFeatures(pickRay, viewer.scene);
- * if (!Cesium.defined(featuresPromise)) {
- *     console.log('No features picked.');
- * } else {
- *     Cesium.when(featuresPromise, function(features) {
- *         // This function is called asynchronously when the list if picked features is available.
- *         console.log('Number of features: ' + features.length);
- *         if (features.length > 0) {
- *             console.log('First feature name: ' + features[0].name);
- *         }
- *     });
- * }
  */
-ImageryLayerCollection.prototype.pickImageryLayerFeatures = function (
+ImageryLayerCollection.prototype.pickImageryLayersHelper = function (
```

Comment:
In most cases helper functions don't need to be added to the class, they can just exist as local functions in the file.